hcl2template/types.packer_config.go

```diff
@@ -367,17 +456,25 @@ func (cfg *PackerConfig) GetBuilds(opts packer.GetBuildsOptions) ([]packer.Build
 			generatedPlaceholderMap := packer.BasicPlaceholderData()
 			if generatedVars != nil {
 				for _, k := range generatedVars {
-					generatedPlaceholderMap[k] = fmt.Sprintf("Build_%s. "+
-						common.PlaceholderMsg, k)
+					generatedPlaceholderMap[k] = ""
 				}
 			}
+			buildVariables, _ := setBuildVariables(generatedPlaceholderMap).Values()
```

Comment:
I'm worried about these values if we decide to have subvalues, something like: `build.bag.something`, this placeholder will default `build.bag` and `something` could cause and error. At the same time, I'm uncertain wether the `unknown value`  from go-cty will solve this issue.

Reply:
Later it's good! Otherwise, we will overload our brains and overengineer the code for something we don't know yet if we are going to need it. :|

Reply:
We likely won't have that kind of variable as long as we're supporting the json templates as equal templates. So having to worry about engineering this is probably at least a year or two out. I'd leave it as is for now.